src/components/sections/Families/HowItWorks.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { ReactElement } from "react";
+import HowItWorksCard from "src/components/cards/HowItWorksCard";
+import LocateLovedOne from "src/assets/HowItWorks/LocateLovedOne.png";
+import SendContent from "src/assets/HowItWorks/SendContent.png";
+import Sample from "src/assets/HowItWorks/Sample.png";
+import DownloadButton from "src/components/buttons/DownloadButton";
+
+//TODO here
+export default function HowItWorks(): ReactElement {
```

Comment:
also i noticed some (but not all) of the components / pages are typed as `ReactElement`. Is this intentional, or should they all be `ReactElement` type?

Reply:
lol legacy code not typed
im using a sick VSCode pluging that does scaffolding for me and this is apparently the most up-to-date way to declare functional components

Reply:
at some point we're gonna scrap everything we have on landing page, so I don't think we need to refactor this rn